src/chainstate/burn/db/sortdb.rs

```diff
@@ -3888,6 +3972,36 @@ impl SortitionDB {
         )
     }
 
+    /// Get the list of Peg-Out Request operations processed in a given burnchain block.
+    /// This will be the same list in each PoX fork; it's up to the Stacks block-processing logic
+    /// to reject them.
+    pub fn get_peg_out_request_ops(
+        conn: &Connection,
+        burn_header_hash: &BurnchainHeaderHash,
+    ) -> Result<Vec<PegOutRequestOp>, db_error> {
+        query_rows(
+            conn,
+            "SELECT * FROM peg_out_requests WHERE burn_header_hash = ?",
+            &[burn_header_hash],
+        )
+    }
+
+    /// Get the list of Peg-Out Fulfill operations processed in a given burnchain block.
+    /// This will be the same list in each PoX fork; it's up to the Stacks block-processing logic
+    /// to reject them.
+    pub fn get_peg_out_fulfill_ops(
+        conn: &Connection,
+        burn_header_hash: &BurnchainHeaderHash,
+    ) -> Result<Vec<PegOutFulfillOp>, db_error> {
+        query_rows(
+            conn,
+            "SELECT * FROM peg_out_fulfillments WHERE burn_header_hash = ?",
+            &[burn_header_hash],
+        )
+    }
+
+    // TODO(3493): Query functions for peg out request and peg out fulfill
```

Comment:
Is this `TODO` satisfied by the above?

Reply:
Yes, thanks for spotting. I'll remove the comment.

Reply:
:heavy_check_mark: